pkg/controller/node/nodecontroller.go

```diff
@@ -274,6 +303,20 @@ func generateCIDRs(clusterCIDR *net.IPNet, num int) sets.String {
 	return res
 }
 
+// translateCIDRs does translating pod CIDRs index to pod CIDRs that could be
+// assigned to node. It will also check for overflow which make sure CIDR is valid
+func translateCIDRs(clusterCIDR *net.IPNet, num int) string {
+	cidrIP := clusterCIDR.IP.To4()
+	b1 := (num >> 8) + int(cidrIP[1])
+	b2 := (num % 256) + int(cidrIP[2])
+	if b2 > 255 {
+		b2 = b2 % 256
+		b1 = b1 + 1
+	}
+	res := fmt.Sprintf("%d.%d.%d.0/24", cidrIP[0], b1, b2)
```

Comment:
Add the same TODO as in generateCIDRs: `// TODO: Make the CIDRs configurable.`

Also delete generateCIDRs, as it should be unused now.